Package.swift

```diff
@@ -33,19 +33,17 @@ func resolveTargets() -> [Target] {
     
     let baseTargets: [Target] = [
         .target(name: "Purchases",
-                dependencies: ["PurchasesSwift"],
+                dependencies: ["PurchasesCoreSwift"],
```

Comment:
I renamed this dependency because it's meant to have the core stuff and be private (well, as private as open source gets), `PurchasesSwift` makes it sound like what you'd use if your project is built in swift (like `CocoaLumberJackSwift`).